libraries/AP_Mount/SoloGimbal_Parameters.cpp

```diff
@@ -115,7 +115,9 @@ void SoloGimbal_Parameters::set_param(gmb_param_t param, float value) {
     char tmp_name[MAVLINK_MSG_PARAM_SET_FIELD_PARAM_ID_LEN+1] {};
     strncpy(tmp_name, get_param_name(param), MAVLINK_MSG_PARAM_SET_FIELD_PARAM_ID_LEN);
 
-    mavlink_msg_param_set_send(_chan, 0, MAV_COMP_ID_GIMBAL, tmp_name, _params[param].value, MAV_PARAM_TYPE_REAL32);
+    if (HAVE_PAYLOAD_SPACE(_chan, PARAM_SET)) {
```

Comment:
are you supposed to be setting _last_request_ms if there's no space and it doesn't send?